File: app/src/main/java/ch/unstable/ost/preference/ContentSettingsFragment.java
package ch.unstable.ost.preference;

import android.os.Bundle;
import android.preference.PreferenceFragment;
import androidx.annotation.Nullable;

import ch.unstable.ost.R;


public class ContentSettingsFragment extends PreferenceFragment {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.pref_content);
    }
}

File: app/src/main/java/ch/unstable/ost/preference/ContentSettingsFragment.kt
package ch.unstable.ost.preference

import android.os.Bundle
import android.preference.PreferenceFragment
import ch.unstable.ost.R

class ContentSettingsFragment : PreferenceFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.pref_content)
    }
}